apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -296,13 +296,12 @@ function AddedSectionsGrid() {
 
     return (
         <Box display="flex" flexDirection="column" gap={1}>
-            <Box display="flex" width={1}>
+            <Box display="flex" width={1} position="absolute" zIndex="2">
                 <CopyScheduleButton />
                 <ClearScheduleButton />
                 <ColumnToggleButton />
             </Box>
-
-            <Box>
+            <Box style={{ marginTop: 45 }}>
```

Comment:
```suggestion
            <Box style={{ marginTop: 50 }}>
```
The course pane's button row takes up a space of 50px, so we should keep that consistent here.